code/modules/events/brain_trauma.dm

```diff
@@ -1,3 +1,5 @@
+//SKYRAT EDIT REMOVAL BEGIN - EVENTS
```

Comment:
Let's not comment this out, instead change the possible events to 0. Removing code like this is bad.

Reply:
How is it bad? It's totally unused, you dont want admins to give a random trauma either.

Reply:
It's still part of the game and we shouldn't really be full removing TG stuff, only making it not spawn in game. WE should also be able to trust admins with this